docs/edit-tool.md

```diff
@@ -154,6 +155,18 @@ In Addtional to the default settings for a tool, you can specify the following o
 
 ![Route Planner Panel](route-planner.png)
 
+## Geomark
+
+The Geomark tool allows creating and loading a geomark, which is an area of interest that is stored in a service and can be shared by a URL. A geomark can be created in SMK by drawing one or more shapes and saving the drawing as a geomark. A geomark can also be loaded by its URL.
+
+### Geomark Service URL
+
+`Geomark Service URL` is the address of the Geomark service used. The production service URL is used as a default, but a user may specify an alternative service URL.
```

Comment:
is the address if of the Geomark web service.

Reply:
Done